src/index.ts

```diff
@@ -181,9 +181,49 @@ export const handleFormEvent =
     )
   }
 
+export const handleFormEvent =
+  (manager: Manager, settings: ComponentSettings) => async (event: MCEvent) => {
+    const { client, payload } = event
+    const { formId, ...restPayload } = payload
+    const { accountId } = settings
+    const url = `https://api.hsforms.com/submissions/v3/integration/submit/${accountId}/${formId}`
+
+    const data: any = {
+      fields: Object.entries(restPayload).map(field => {
+        return {
+          name: field[0],
+          value: field[1],
+        }
+      }),
+      context: {
+        pageUri: client.url.href,
+        pageName: client.title,
+        ipAddress: client.ip,
+      },
+    }
+
+    const visitorCookieName = 'hubspotutk'
```

Comment:
Looks like this is already defined in line 47. can we just move that to the global scope so that we won't have to redefine it?